e2e/wdio/platform/tests/search.e2e-spec.ts

```diff
@@ -1,14 +1,143 @@
+            click(searchPo.clearSearchIcon);
+        }
+        expect(getText(searchPo.cozySearchResult, 0)).not.toContain('test');
+        expect(getText(searchPo.compactSearchResult, 0)).not.toContain('test');
+        expect(getText(searchPo.cozyWithCategoriesSearch, 0)).not.toContain('test');
+        expect(getText(searchPo.compactWithCategoriesSearch, 0)).not.toContain('test');
+        expect(getText(searchPo.cozyWithDataSourceSearch, 0)).not.toContain('test');
+
+        expect(getText(searchPo.cozySearchResult, 1)).not.toContain('test');
+        expect(getText(searchPo.compactSearchResult, 1)).not.toContain('test');
+        expect(getText(searchPo.cozyWithCategoriesSearch, 3)).not.toContain('test');
+        expect(getText(searchPo.compactWithCategoriesSearch, 3)).not.toContain('test');
+        expect(getText(searchPo.cozyWithDataSourceSearch, 3)).not.toContain('test');
+    });
+
+
+    it('should have autosuggestion after one latter', () => {
+        const arrLength = getElementArrayLength(searchPo.searchFields);
+        for (let i = 0; arrLength > i; i++) {
+            // value without suggestion
+            setValue(searchPo.searchFields, 'ea', i);
+            waitElementToBePresentInDOM(searchPo.autosuggestionItems);
+            waitForElDisplayed(searchPo.autosuggestionItems);
+            getTextArr(searchPo.autosuggestionItems).forEach((suggestionItemText) => {
+                expect(suggestionItemText).toContain('ea');
+            });
+            click(searchPo.clearSearchIcon);
+        }
+    });
+
+    it('should compact be smaller than cozy', () => {
+        const defaultCozySize = getElementSize(searchPo.searchFields, 0, 'height');
+        const defaultCompactSize = getElementSize(searchPo.searchFields, 1, 'height');
+        const withCategoryCozySize = getElementSize(searchPo.searchFields, 2, 'height');
+        const withCategoryCompactSize = getElementSize(searchPo.searchFields, 3, 'height');
+
+        expect(defaultCozySize).toBeGreaterThan(defaultCompactSize);
+        expect(withCategoryCozySize).toBeGreaterThan(withCategoryCompactSize);
+    });
+
+    it('should be able so set category if this option enabled', () => {
+        const arrLength = getElementArrayLength(searchPo.searchCategoryBtn);
+        for (let i = 0; arrLength > i; i++) {
+            click(searchPo.searchCategoryBtn, i);
+            click(searchPo.autosuggestionItems);
+            click(searchPo.searchIcons, i + 2);
+        }
+        expect(getText(searchPo.cozyWithCategoriesSearch, 1)).toContain('red');
+        expect(getText(searchPo.cozyWithCategoriesSearch, 3)).toContain('red');
+
+        expect(getText(searchPo.compactWithCategoriesSearch, 1)).toContain('red');
+        expect(getText(searchPo.compactWithCategoriesSearch, 3)).toContain('red');
+
+        expect(getText(searchPo.cozyWithDataSourceSearch, 1)).toContain('red');
+        expect(getText(searchPo.cozyWithDataSourceSearch, 3)).toContain('red');
```

Comment:
maybe you should create a variable for the contain value in case the examples change. Since that's happened to me a few times already.